cves/kernel/CVE-2012-6545.yml

```diff
@@ -55,7 +55,8 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Some memory was not being initialized correctly, which meant the memory was being leaked. A well thought out program could use this to read sensitive information.
```

Comment:
How does this memory leak affect the functionality of the program itself/ what areas in CIA does this affect?

Reply:
Good idea, I'll add the CIA info.